src/hero.js

```diff
@@ -3,35 +3,33 @@ const template = document.createElement('template');
 template.innerHTML = `
   <div id="theClub" class="hero">
     <div class="hero__image the-club">
-      <h2 class="hero__title">
-      The<br>Breakfast<br>Club</h2>
-      <h3></h3>
-      <p>prepared with only natural </p>
+      <h2 class="hero__title">The Club</h2>
+      <p class="hero__content">prepared with only natural </p>
     </div>
   </div>
 
   <div id="theBreakfast" class="hero offscreen">
     <div class="hero__image the-breakfast">
-      <h2 class="hero__title">Breakfast</h2>
-      <h3></h3>
-      <p>hello with only natural </p>
+      <h2 class="hero__title">The Breakfast</h2>
+      <p class="hero__content">hello with only natural</p>
     </div>
   </div>
 
   <div id="ourLocations" class="hero offscreen">
     <div class="hero__image our-locations">
-      <h2 class="hero__title">Breakfast</h2>
-      <h3></h3>
-      <p>ready with only natural </p>
+      <h2 class="hero__title">Our Locations</h2>
+      <p class="hero__content">ready with only natural </p>
     </div>
   </div>
 `;
 
-function hero() {
-  const hero = document.createElement('div');
-  hero.innerHTML = template.innerHTML;
+class Hero {
+  hero() {
+    const hero = document.createElement('div');
+    hero.innerHTML = template.innerHTML;
 
-  return hero;
+    return hero;
+  }
 }
 
-export default hero;
+export default Hero;
```

Comment:
- [ ] [REQUIRED] The contents of each tab are stored in a separate module each
- [ ] [REQUIRED] Each module should export a function that creates a `<div>` element, adds the appropriate content and styles to that element, and then appends itself to the DOM
![image](https://user-images.githubusercontent.com/73254040/99837519-54227880-2b2d-11eb-8ae4-3a70727e1012.png)
[Source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)